data-pipeline-flink/user-cache-updater-2.0/src/test/scala/org/sunbird/dp/spec/UserCacheUpdatetStreamTaskSpecV2.scala

```diff
@@ -70,10 +70,10 @@ class UserCacheUpdatetStreamTaskSpecV2 extends BaseTestSpec {
   def setupRedisTestData(jedis: Jedis) {
 
     // Insert user test data
-    jedis.hmset("user-3", EventFixture.userCacheDataMap3)
-    jedis.hmset("user-4", EventFixture.userCacheDataMap4)
-    jedis.hmset("user-9", EventFixture.userCacheData9)
-    jedis.hmset("user-11", EventFixture.userCacheData11)
+    jedis.hmset("user:user-3", EventFixture.userCacheDataMap3)
+    jedis.hmset("user:user-4", EventFixture.userCacheDataMap4)
+    jedis.hmset("user:user-9", EventFixture.userCacheData9)
+    jedis.hmset("user:user-11", EventFixture.userCacheData11)
```

Comment:
Update all test cases to read the prefix from "config.userStoreKeyPrefix" variable

Reply:
Done